prysm/Onyx(v0.12.1)/config.yaml

```diff
@@ -0,0 +1,61 @@
+FAR_FUTURE_EPOCH: 18446744073709551615
+BASE_REWARDS_PER_EPOCH: 4
+DEPOSIT_CONTRACT_TREE_DEPTH: 32
+MAX_COMMITTEES_PER_SLOT: 64
+TARGET_COMMITTEE_SIZE: 128
+MIN_PER_EPOCH_CHURN_LIMIT: 4
+CHURN_LIMIT_QUOTIENT: 65536
+SHUFFLE_ROUND_COUNT: 90
+MIN_GENESIS_ACTIVE_VALIDATOR_COUNT: 16384
+MIN_GENESIS_TIME: 1578009600
+MIN_GENESIS_DELAY: 86400
```

Comment:
This is not a v0.12.1 config value, it should be `GENESIS_DELAY: 172800` Can you check the Prysm config?

Reply:
That's right. It should be 172800. Thanks